### ISSUE TITLE
 X shape is not announced by NVDA.

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#13636]
[NotepadWithShapes.txt](https://github.com/nvaccess/nvda/files/8558718/NotepadWithShapes.txt)

### Steps to reproduce:
1. Turn on NVDA.
2. Open the attached Notepad.
3. Navigate the edit pane and observed that NVDA is not narrating the 'X' shape. 

### Actual behavior:
 X shape is not announced by NVDA.

### Expected behavior:
 X shape should be announced by NVDA.

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
Version 2021.3.5 (eSpeak NG Synthesizer)

#### Windows version:
OS: Windows 11 Version 21H2 (Build 22000.613)

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No

**Comment by josephsl** [comment]
Hi, which character is NVDA not announcing? At least with latest alpha build (25262), NVDA with eSpeak NG enabled does announce black square and filled up pointing triangle characters. I don’t think Narrator is announcing any characters in the attached document (I opened it on Windows 11 nickel build 22598 with Notepad 11.2203 installed). Thanks.

**Comment by CyrilleB79** [comment]
The character which is not announced is the following: ☓
It is the first character in the file.

It is character unicode 9747, i.e. u+2613 in hexa.
It is not part of CLDR.

There are probably other tickets to increase the number of symbols that are announced by NVDA so this should probably be commented there.

@msftedad, could you please indicate in which circumstance you have decided to open this ticket? In which real-life example have you found this character? That would help understand why it is important to have it reported. Thanks.

and it is called SALTIRE in the Unicode norm.

**Comment by Brian1Gaff** [comment]
says here symbol 2 6 1 3

No idea what that is.
 Brian

***@***.***
Sent via blueyonder.
Please address personal E-mail to:-
***@***.***, putting 'Brian Gaff'
in the display name field.
Newsgroup monitored: alt.comp.blind-users
----- Original Message -----

**Comment by msftedad** [comment]
@CyrilleB79, Thank you for acknowledging the issue. We have observed this character in Power BI under this path: 'Visualizations' pane-> Visual-> Markers-> Shape-> Marker Shape drop-down. (Adding the screenshot for reference)
![image](https://user-images.githubusercontent.com/93735775/165648606-efc8eb8a-8bdd-40fc-9a54-d6ebdde46651.png)

And JAWS is announcing this character as "SALTIRE" but not NVDA.

**Comment by seanbudd** [comment]
See also: #9158
